lib/rspec/matchers/built_in/match.rb

```diff
@@ -67,27 +67,17 @@ def match_captures(expected, actual)
         end
       end
 
-      # @api private
-      # Used to wrap match data and make it reliable for 1.8.7
-      class ReliableMatchData
+      # @private
+      class MatchData
```

Comment:
Given the comment, you should be able to remove this entire class and use match_data directly now, otherwise revert.

Reply:
Aha, nice catch!